client/src/app/site/agenda/components/agenda-item-list/agenda-item-list.component.ts

```diff
@@ -179,7 +188,7 @@ export class AgendaItemListComponent extends BaseListViewComponent<ViewAgendaIte
                     follow: [
                         {
                             idField: 'content_object_id',
-                            follow: [SPEAKER_BUTTON_FOLLOW],
+                            follow: [SPEAKER_BUTTON_FOLLOW, SUBMITTER_FOLLOW],
```

Comment:
@tsiegleauq I didn't remembered, that I already had this implemented :D It wasn't in use (The warning I removed was never triggered), this is the first case.